src/models/Studio.ts

```diff
@@ -1,4 +1,5 @@
-import { promises as fPromises } from 'fs';
+import { existsSync ,promises as fPromises } from 'fs';
+import {SpecificationFileNotFound} from '../errors/specification-file';
```

Comment:
```suggestion
import { existsSync, promises as fPromises } from 'fs';
import { SpecificationFileNotFound } from '../errors/specification-file';
```

super not relevant formatting improvement, rest is good